charts/plane-ce/templates/workloads/worker.deployment.yaml

```
@@ -23,12 +23,21 @@ spec:
         stdin: true
         tty: true
         resources:
+          {{- if eq (include "enable.hpa" . | trim) "true" }}
+          requests:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)